### ISSUE TITLE
Print module repeatedly is disabled with each boa / or drupal core update.

### ISSUE THREAD

**Issue by EdNett** [omega8cc/boa#710]
For a few months now I have noticed that the print module (provided by o_contrib_seven) keeps getting deactivated with each boa update OR with each drupal core update. I believe this is happening only on selected sites. I am going to try to debug this myself and will post further info. When this occurs, the settings are kept, and all I have to do is re-enable the module for it to work as before.

For example I just upgraded a static platform to 7.37 core, and noticed that the module (all 4 or 5 submodules) was completely disabled.

barracuda_log.txt at:
https://gist.github.com/EdNett/26edcd256f5e7ac87681

octopus_log.txt:
https://gist.github.com/EdNett/f11756cf6af93b5daedc

barracuda_cnf at:
https://gist.github.com/EdNett/159c4f2e886399ccc4c3

octopus_cnf at:
https://gist.github.com/EdNett/abdcc90d96ed19d3c250

**Comment by EdNett** [comment]
This problem (print modules becoming automagically disabled) also occurred recently when we manually moved boa hosted drupal 7 static platform sites (using a name like "fred3" as the octopus instance name) from one server to another. The print modules were enabled on the site, but after we moved the sites to the other server (creating the sites on the new server first, then writing over th "files" directory and importing the old database.sql file into the new database), they were disabled.  Could the verify task run be causing this?
